src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -413,6 +413,22 @@ for local network permission. The permission can also be allowed by enabling
 </li>
 </ol>
 
+## Apple Silicon (`arm64` Macs)
+
```

Comment:
Do we need to mention any Flutter framework version or it doesn't matter?

Reply:
Nope, this would be needed for any version since none of them support arm64 simulators.